README.md

```diff
@@ -19,9 +19,408 @@ BiggerQuery lets you implement complex stuff (the Allegro experimentation platfo
 
 `pip install biggerquery`
 
+`pip install biggerquery[beam]`(if you want to use the Apache Beam)
+
 ## Compatibility
 
-BiggerQuery is compatible with Python >= 3.7.
+BiggerQuery is compatible with Python >= 3.6.
+
+## Cheat sheet
+
+### Setup
+
+We recommend using the Jupyter Lab to go through the examples. You can also run the examples as scripts, or from
+your own Jupyter notebook. In those cases, you can authorize using `pydata_google_auth`(look at the example below) or [Google sdk](https://cloud.google.com/sdk/docs/quickstarts).
+ 
+Inside this repository you can find the file named 'MilitaryExpenditure.csv'. Use the script below to load the csv to the BigQuery table.
+You will use the created table to explore the BiggerQuery methods.
+
+First of all, install the dependencies:
+
+`pip install biggerquery`
+
+`pip install pydata_google_auth`
+
+Then, fill up the PROJECT_ID and DATA_PATH:
+
+```python
+PROJECT_ID = 'put-you-project-id-here'
+DATA_PATH = '/path/to/json/file/311_requests.csv'
+
+import biggerquery as bgq
+import pydata_google_auth
+import pandas as pd
+
+credentials = pydata_google_auth.get_user_credentials(['https://www.googleapis.com/auth/bigquery'])
+
+dataset = bgq.Dataset(
+    project_id=PROJECT_ID,
+    dataset_name='external_data',
+    credentials=credentials)
+
+df = pd.read_csv(DATA_PATH, dtype={
+    'street_number': str,
+    'state_plane_x_coordinate': str
+})
+
+load_table = dataset.load_table_from_dataframe('311_requests', df, partitioned=False)
+load_table.run()
+```
+
+### Authorize with a GCP user account
```

Comment:
Is this some kind of duplication? I dont see much difference between this paragraph and 50-59 lines